server/app/app.py

```diff
@@ -69,6 +72,7 @@ def main():
     parser.add_argument("--title", "-t", help="Title to display -- if this is omitted the title will be the name "
                                               "of the directory from the data_directory arg")
     parser.add_argument("--port", help="Port to run server on.", type=int, default=5005)
+    parser.add_argument("--no-open", help="Do not launch the webbrowser.", action='store_true')
```

Comment:
Often times, I find it helpful to reason parameters like this as:

`parser.add_argument("--no-open", help="Do not launch the webbrowser.", action="store_false", dest="open_browser")`

then your code would be:

```
if args.open_browser:
    ...
```

Rationale: you write the crufty inverted logic once for `parser.add_argument`, and the rest of code is much easier to read.  Since you only use `no_open` once, it's nbd.

I would fix the inconsistent string quote style though. :)

Reply:
That is much cleaner. 🤦‍♀️ on the string quote again...